src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -91,7 +90,7 @@ def transform_stacks_list(result):
                 resources += reslist['id'] + ","
 
             res = OrderedDict([('Name', r['name']), ('State', r['provisioningState']),
-                              ('Last Modified', r['systemData']['lastModifiedAt']), ('Resource IDs', resources[:-1])])
```

Comment:
https://github.com/Azure/deployment-stacks/issues/112

We want to remove resource and deployment Ids from table output

Reply:
May I ask if this change will cause a breaking change for customers?

Reply:
We have not made the announcement yet, so very few customers are currently using stacks CLI and since this is table output it should not cause breaking change.